CONTRIBUTING.md

```diff
@@ -36,12 +36,15 @@ Follow these steps to build, run, and test the repository:
 
 #### Building
 
+If you have the ability to run powershell, you can invoke "build.ps1" from the root of the repo. If not, the following steps will get build going for you as well:
```

Comment:
I would prefer it named `init.ps1` as the script doesn't actually invoke a build. It would need to additionally run `npm run compileDev` or similar to build the extension.

Reply:
Done. 

As an aside, would it help having a build script?

Reply:
I don't think so. In web projects, `npm run` is a fairly universal way of running these build tasks and is only a single command. However, I am open to reviewing a PR if we thought it would improve onboarding.